projects/storefrontapp/src/app/spartacus/features/product-configurator-textfield-feature.module.ts

```diff
@@ -0,0 +1,28 @@
+import { NgModule } from '@angular/core';
+import { provideConfig } from '@spartacus/core';
+import { configuratorTranslations } from '@spartacus/product-configurator/common/assets';
+import {} from '@spartacus/product-configurator/rulebased/root';
+import {
+  PRODUCT_CONFIGURATOR_TEXTFIELD_FEATURE,
+  TextfieldConfiguratorRootModule,
+} from '@spartacus/product-configurator/textfield/root';
+
+@NgModule({
+  imports: [TextfieldConfiguratorRootModule],
+  providers: [
+    provideConfig({
+      featureModules: {
+        [PRODUCT_CONFIGURATOR_TEXTFIELD_FEATURE]: {
+          module: () =>
+            import('@spartacus/product-configurator/textfield').then(
+              (m) => m.TextfieldConfiguratorModule
+            ),
+        },
+      },
+      i18n: {
+        resources: configuratorTranslations,
```

Comment:
Repeating same translations both in textfield feature module and rulebased feature module makes me wonder whether we should split those translations in between those features.

I'm happy to know your opinion @ChristophHi .

But it's definitely not a blocker for this pr!

Reply:
Hi @Platonn , I checked that: A split does not make sense (as textfield only has 2 translatable texts, which it shares with rulebased). Also we have 60 texts in total, so we don't harm the textfield scenario when loading all of them. 
Let's revisit this when we introduce other configurators...